app/src/main/java/com/hedvig/app/feature/home/ui/claimstatus/data/RelatedContractTypeData.kt

```diff
@@ -0,0 +1,16 @@
+package com.hedvig.app.feature.home.ui.claimstatus.data
+
+import androidx.compose.runtime.Composable
+import com.hedvig.android.owldroid.graphql.HomeQuery
+
+@JvmInline
+value class RelatedContractTypeData(val text: String) {
```

Comment:
How much mileage are we getting from having this be a `value class` wrapping a `String`? 🙂